hrms/hr/doctype/expense_claim/expense_claim.json

```diff
@@ -221,7 +221,6 @@
    "oldfieldname": "company",
    "oldfieldtype": "Link",
    "options": "Company",
-   "read_only": 1,
```

Comment:
fetch from also executes on the server side, so I think selected company will get overwritten. Can you check once? If thats happening you will also have to enable `fetch_if_empty` on the field